scripts/utils.mjs

```diff
@@ -7,7 +7,11 @@ import { readFile } from 'node:fs/promises';
 import { join, resolve } from 'node:path';
 import pemfile from 'pem-file';
 
-const SNS_JSON_FILE = join(process.cwd(), 'src', 'frontend', 'src', 'env', 'tokens.sns.json');
+const DATA_DIR = join(process.cwd(), 'src', 'frontend', 'src', 'env');
+
+export const CK_ERC20_JSON_FILE = join(DATA_DIR, 'tokens.ckerc20.json');
```

Comment:
should we create a `constants.mjs` or `env.mjs` or something? I would spontaneously not search in a utils file since we generally use constants module. WDYT?

Reply:
Yes, makes sense! Done!

I just didn't move this, because it seems like a non-constants, if you know what i mean... WDYT?

`export const SNSES = JSON.parse((await readFile(SNS_JSON_FILE)).toString(), jsonReviver);`

Reply:
Agree!